webauthn-authenticator-rs/examples/softtoken/main.rs

```diff
@@ -0,0 +1,43 @@
+extern crate tracing;
```

Comment:
Some of these commands are actually useful and we should move them to authenticator-cli instead soon.

Reply:
Agreed